src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -874,6 +876,61 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		return false;
 	}
 
+	/**
+	 * Refresh Azure access token if it's expired.
+	 * @param uri connection uri
+	 * @returns true if no need to refresh or successfully refreshed token
+	 */
+	public async refreshAzureAccountTokenIfNecessary(uri: string): Promise<boolean> {
+		let profile = this._connectionStatusManager.getConnectionProfile(uri);
```

Comment:
general comments, if a variable is not supposed to be changed, please use const instead of let.

Reply:
updated